dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/TableDefinition.kt

```diff
@@ -377,6 +379,8 @@ class TableDefinition(manager: ProcessorManager, element: TypeElement) : BaseTab
 
         getAllColumnPropertiesMethod.initializer("new \$T[]{\$L}", ClassNames.IPROPERTY, getPropertiesBuilder.build().toString())
         typeBuilder.addField(getAllColumnPropertiesMethod.build())
+        getStarPropertyMethod.initializer("new \$T(\$T.class, \$T.rawBuilder(\"*\").build())", ClassNames.PROPERTY, elementClassName, ClassNames.NAMEALIAS)
```

Comment:
you can use a `\$S` in the place of the current `\"*\"`, then add `"*"` to the param list.

It should always have `.withTable()` since the default `select()` is `SELECT *` it would be a source of subtle bugs if the query builder usage was `select(Something_Table.STAR)` where as if it always has the table qualifier the output query will read just like the  query dsl `select(Something_Table.STAR)` becomes ```SELECT `Something`.*```